packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -15,10 +15,12 @@ class BuildApkCommand extends BuildSubCommand {
     usesPubOption();
 
     argParser
-      ..addFlag('preview-dart-2', negatable: false,  hide: !verboseHelp)
+      ..addFlag('preview-dart-2', defaultsTo: true, hide: !verboseHelp)
```

Comment:
here and elsewhere: we should add help to these `preview-dart-2` flags

Reply:
Done.